index.bs

```diff
@@ -2289,10 +2297,33 @@ during registration.
     :   {{AuthenticatorAttestationResponse/getTransports()}}
     ::  This operation returns the value of {{AuthenticatorAttestationResponse/[[transports]]}}.
 
+    :   {{AuthenticatorAttestationResponse/getAuthenticatorData()}}
+    ::  This operation returns the [=authenticator data=] contained within {{AuthenticatorAttestationResponse/attestationObject}}. See [[#sctn-public-key-easy]].
+
+    :   {{AuthenticatorAttestationResponse/getPublicKey()}}
+    ::  This operation returns a string that contains the base64-encoded, DER [=SubjectPublicKeyInfo=] of the new credential, or an empty string if this is not available. See [[#sctn-public-key-easy]].
+
     :   <dfn>\[[transports]]</dfn>
     ::  This [=internal slot=] contains a sequence of zero or more unique {{DOMString}}s in lexicographical order. These values are the transports that the [=authenticator=] is believed to support, or an empty sequence if the information is unavailable. The values SHOULD be members of {{AuthenticatorTransport}} but [=[RPS]=] MUST accept unknown values.
 </div>
 
+#### Easily accessing credential data #### {#sctn-public-key-easy}
+
+Every user of the [[#sctn-createCredential|create method]] will need to parse and store the returned public key in order to verify future assertions. However, the public key is in [[!RFC8152]] (COSE) format, inside the [=credentialPublicKey=] member of the [=attestedCredentialData=], inside the [=authenticator data=], inside the [=attestation object=] from {{AuthenticatorAttestationResponse/attestationObject}}. Users that wish to use [=attestation=] MUST do the work of parsing that out because that copy of the public-key is the one that the [=authenticator=] has [signed](#signing-procedure). However, many valid uses of WebAuthn do not need [=attestation=]. For those uses, user agents can do the work of parsing, expose the [=authenticator data=] directly, and translate the public key into a more convenient format.
+
+The {{AuthenticatorAttestationResponse/getPublicKey()}} operation thus returns the credential public key as a [=SubjectPublicKeyInfo=]. After base64 decoding, this format can, for example, be passed to Java's `java.security.spec.X509EncodedKeySpec`, .NET's `System.Security.Cryptography.ECDsa.ImportSubjectPublicKeyInfo`, or Go's `crypto/x509.ParsePKIXPublicKey`.
+
+Use of this function does impose some limitations: by using {{PublicKeyCredentialCreationOptions/pubKeyCredParams}}, a [=[RP]=] can negotiate with the [=authenticator=] to use public-key algorithms that the user agent may not understand. However, if it does so, the user agent will not be able translate the resulting public key into [=SubjectPublicKeyInfo=] format and the return value of {{AuthenticatorAttestationResponse/getPublicKey()}} will be an empty string.
```

Comment:
```suggestion
Use of {{AuthenticatorAttestationResponse/getPublicKey()}} does impose some limitations: by using {{PublicKeyCredentialCreationOptions/pubKeyCredParams}}, a [=[RP]=] can negotiate with the [=authenticator=] to use public key algorithms that the user agent may not understand. However, if it does so, the user agent will not be able translate the resulting [=credential public key=] into [=SubjectPublicKeyInfo=] format and the return value of {{AuthenticatorAttestationResponse/getPublicKey()}} will be an empty string.
```

Reply:
There's also a missing "to" in "be able translate":

```suggestion
Use of {{AuthenticatorAttestationResponse/getPublicKey()}} does impose some limitations: by using {{PublicKeyCredentialCreationOptions/pubKeyCredParams}}, a [=[RP]=] can negotiate with the [=authenticator=] to use public key algorithms that the user agent may not understand. However, if it does so, the user agent will not be able to translate the resulting [=credential public key=] into [=SubjectPublicKeyInfo=] format and the return value of {{AuthenticatorAttestationResponse/getPublicKey()}} will be an empty string.
```